local-cli/core/ios/findProject.js

```diff
@@ -19,7 +19,7 @@ const GLOB_PATTERN = '**/*.xcodeproj';
 /**
  * Regexp matching all test projects
```

Comment:
Since i am not the original author of this code i can not be certain what the intention was